plugins/inputs/gtfs/gtfs.go

```diff
@@ -0,0 +1,305 @@
+// Package gtfs is an input plugin for collecting GTFS-realtime data.
```

Comment:
Remove.

Reply:
A package-level comment summarizing the package's purpose is [idiomatic Go](https://blog.golang.org/godoc). I disagree this should be removed.

Reply:
While I agree that this is standard in Go programs, this is a plugin to telegraf and I think most (if not all) other plugins do not have such a comment. I like to have things identical if there is no actual need to deviate from "the-usual" as this eases reviews over a large number of plugins. If this comment is really important to you, I will take to the core-devs on how to handle these. Just let me know.